src/plugins/es_ui_shared/static/forms/hook_form_lib/components/use_field.test.tsx

```diff
@@ -64,6 +64,93 @@ describe('<UseField />', () => {
     });
   });
 
+  describe('validation', () => {
+    let formHook: FormHook | null = null;
+
+    beforeEach(() => {
+      formHook = null;
+    });
+
+    const onFormHook = (form: FormHook) => {
+      formHook = form;
+    };
+
+    const getTestComp = (fieldConfig: FieldConfig) => {
+      const TestComp = ({ onForm }: { onForm: (form: FormHook) => void }) => {
+        const { form } = useForm<any>();
+
+        useEffect(() => {
+          onForm(form);
+        }, [onForm, form]);
+
+        return (
+          <Form form={form}>
+            <UseField path="name" config={fieldConfig} data-test-subj="myField" />
+          </Form>
+        );
+      };
+      return TestComp;
```

Comment:
nit: `TestComp` variable is not necessarily needed here.

Reply:
True, but sometimes adding the var makes it a little bit clearer when scanning the file than directly returning a function. I guess it depends on my level of fatigue 😊 
In this case, I clearly read what is the component without any extra parsing.